### DISCUSSION TITLE
Does topology logic refresh trigger ConnectionActivedEvent and ConnectionDeactivedEvent?

### DISCUSSION THREAD

**Discussion by BozhongWei** [redis/lettuce#2187] [Q&A]
I haveI found that when the topology logic is refreshed, there will be frequent ConnectionActivedEvent and ConnectionDeactivedEvent。Does topology logic refresh trigger ConnectionActivedEvent and ConnectionDeactivedEvent?

**Comment by mp911de** [comment]
Yes, it does because topology refresh connects and disconnects to individual cluster nodes.
